Core/clim-core/commands/tables.lisp

```diff
@@ -377,17 +359,21 @@ examine the type of the command menu item to see if it is
     (values)))
 
 (defun find-menu-item (menu-name command-table &key (errorp t))
-  (let* ((table (find-command-table command-table))
-         (mem (member menu-name (slot-value table 'menu)
-               :key #'command-menu-item-name :test #'string-equal)))
-    (if mem
-        (values (car mem) command-table)
-        (or (find-if #'(lambda (table)
-                         (find-menu-item menu-name table :errorp nil))
-                     (command-table-inherit-from table))
-            (when errorp
-              (error 'command-not-accessible :command-table-name
-                     (command-table-designator-as-name table)))))))
+  (loop with table = (find-command-table command-table)
+        for item in (slot-value table 'menu)
+        when (string-equal menu-name (command-menu-item-name item))
+          do (return-from find-menu-item
+               (values item command-table)))
```

Comment:
Why `loop` instead of `when-let ((… (find …`?

Reply:
as noted below, this code is not incorrect and immedietely replaced later (as in - it is just a transient state for clarity)

Reply:
OK.